src/items.cpp

```diff
@@ -819,8 +821,29 @@ void Items::parseItemNode(const pugi::xml_node& itemNode, uint16_t id)
 					break;
 				}
 
-				case ITEM_PARSE_DURATION: {
-					it.decayTime = pugi::cast<uint32_t>(valueAttribute.value());
+				case ITEM_PARSE_DURATION:
+				case ITEM_PARSE_DURATIONMIN: {
+					auto split = explodeString(valueAttribute.as_string(), ";");
+					if (split.size() == 2) {
+						it.decayTime = pugi::cast<uint32_t>(split[0].c_str());
+						uint32_t value = pugi::cast<uint32_t>(split[1].c_str());
+						if (it.decayTimeMax) {
+							std::cout << "[Warning - Items::parseItemNode] decayTimeMax is being overwritten to: " << value << std::endl;
+						}
+						it.decayTimeMax = value;
+					}
+					else {
```

Comment:
this whole code is not needed as we are not using ; serparator for min and max
just
it.decayTime = pugi::cast<uint32_t>(valueAttribute.value());

Reply:
I think you hate the separator ahahahaha, but since you're the only one paying attention to me, I'm going to listen to you <3